### ISSUE TITLE
File IO

### ISSUE THREAD

**Issue by legolord208** [elliotchance/c2go#60]
```C
#include <stdio.h>

int main()
{
	printf("Hello World\n");

	FILE *f = fopen("test.txt", "w");
	if (f == NULL)
	{
		printf("Error :(\n");
		return 0;
	}

	fprintf(f, "Hello World\n");

	fclose(f);
}
```

## Converts to:

```Go
func main() {
	fmt.Printf("Hello World\n")
	var f *int64 = fopen("test.txt", "w")

	if f == nil {
		fmt.Printf("Error :(\n")
		return

	}

	fprintf(f, "Hello World\n")
	fclose(f)

}
```

Nice try, but it needs a tiny bit more work ;)

**Comment by elliotchance** [comment]
c2go does not officially support file IO yet. It *sort of* works because some of the file functions in `stdio.h` have had aliases setup to get the compiler working with no actual implementations of those functions yet.

Support for `stdio.h` (which includes those file functions) is coming in the next version [v0.10.0](https://github.com/elliotchance/c2go/milestone/3). Stay tuned... or better, help out :)

**Comment by elliotchance** [comment]
This should be fixed in v0.11.0

**Comment by legolord208** [comment]
Nice work!
